roxie/ccd/ccdserver.cpp

```diff
@@ -1663,7 +1660,10 @@ class CRoxieServerStrandedActivity : public CRoxieServerActivity
         active = 0;
     }
 
-    //This function is pure (But also implemented out of line) to force the derived classes to implement it.
```

Comment:
Looks like it has become separated from the function it was supposed to refer to... I will fix